cmd/prom2json/main.go

```diff
@@ -29,6 +29,7 @@ import (
 func main() {
 	cert := flag.String("cert", "", "certificate file")
 	key := flag.String("key", "", "key file")
+	skipCAcheck := flag.Bool("skipCAcheck", false, "Skip certificate check during tls handshake")
```

Comment:
We don't use camel case in flags. This should be `skip-ca-check`. However, I really want strong wording in there that this is insecure and for testing only. Howe about `accept-invalid-cert` and as the doc string: "Accept any certificate during TLS handshake. Insecure, use only for testing."

I'd call the variable something like `skipServerCertCheck` to make clear this is about the server`s cert, not the cert provided above.